poly.go

```diff
@@ -103,6 +103,7 @@ type Sequence struct {
 	SequenceHashFunction string    `json:"hash_function"`
 	Sequence             string    `json:"sequence"`
 	Features             []Feature `json:"features"`
+	MD5                  [16]byte  `json:"hash"` //MD5 checksum
```

Comment:
Considering that we already have sequence_hash and hash_function can we call this something like `GenbankCheckSum` in both the struct and json output @Koeng101?